tools/mpremote/mpremote/commands.py

```diff
@@ -555,7 +556,7 @@ def _do_romfs_deploy(state, args):
     romfs_filename = args.path
 
     # Read in or create the ROMFS filesystem image.
-    if romfs_filename.endswith(".romfs"):
+    if romfs_filename.endswith((".img", ".romfs")):
```

Comment:
Oh, I thought you wanted to keep `isfile` (which I think is a good idea, because maybe a directory ends with `.romfs` for some reason) and so this would be:
```py
if os.path.isfile(romfs_filename) and romfs_filename.endswith((".img", ".romfs")):`
```

Reply:
Okay applied the above.